api/services/PassportService.js

```diff
@@ -203,11 +203,12 @@ module.exports = class PassportService extends Service {
 
     criteria[fieldName] = identifier
 
-    return this.app.services.FootprintService.find('User', criteria, {populate: 'passports', findOne: true})
+    return this.app.services.FootprintService.find('User', criteria, {populate: 'passports'})
```

Comment:
`FootprintService.find` with findOne option on trailpack-mongoose assumes that find criteria is a MongoId and fail to cast `{email: [value]}` into a valid ObjectId, I think it's a limitation of the trailpack
I'm quite busy for now but I can arrange a fix in the future and then restore the past behavior here